lib/src/quality/apps/text/links.py

```diff
@@ -9,35 +9,50 @@ class Links(BaseModel):
     link: str
 
 @processor()
-def asser_links(df: DF[Links], context: Context):
+def asser_links(df: DF[Links], context: Context) -> pd.DataFrame:
     """
         Check if the links are valid (do not return error status codes)
 
-        Input:
-          DF[Links] - DataFrame with one column named 'link'.
+    ## Input:
+           A dataframe that contains columns:
+                - link: link to be validated
 
-        Output:
-          DF[Links] - DataFrame with one column named 'link' containing
-          valid or invalid links depending on 'filter_links' option.
+    ## Output:
+            The format of dataframe depends on the configuration provided.
 
-        Options:
-          filter_links - If True, returns links which passed assert. Else return
-          error links.
-    """
+            If `filter_links` set to True, the output will be a dataframe
+            with a column named `link` containing valid links.
+
+            Otherwise the output will be a dataframe with a column named
+            `error` containing invalid links.
+
+    ## Configuration:
+        - filter_links(bool, optional): If set to True, will filter the dataframe and exclude invalid links. Default value is False
+
+    ---
+
+    Args:
+        df (DF[Links]): A dataframe with a column named `link` containing links.
```

Comment:
It is obvious that `link` column contains links. Make it more natural:

| A dataframe with a column `link` containing HTTP links to be checked